test/api_integration/apis/saved_objects/bulk_get.ts

```diff
@@ -115,5 +140,130 @@ export default function ({ getService }: FtrProviderContext) {
           expect(resp.body.saved_objects[0].migrationVersion).to.be.ok();
           expect(resp.body.saved_objects[0].typeMigrationVersion).to.be.ok();
         }));
+
+    it('should return 200 with individual responses that include the managed property of each object', async () =>
+      await supertest
+        .post(`/api/saved_objects/_bulk_get`)
+        .send(BULK_REQUESTS_MANAGED)
+        .expect(200)
+        .then((resp) => {
+          const mockDate = '2015-01-01T00:00:00.000Z';
+          resp.body.saved_objects[0].updated_at = mockDate;
+          resp.body.saved_objects[1].updated_at = mockDate;
+          resp.body.saved_objects[2].updated_at = mockDate;
+          resp.body.saved_objects[3].updated_at = mockDate;
+          resp.body.saved_objects[0].created_at = mockDate;
+          resp.body.saved_objects[1].created_at = mockDate;
+          resp.body.saved_objects[2].created_at = mockDate;
+          resp.body.saved_objects[3].created_at = mockDate;
```

Comment:
NIT: be lazy. `for obj in resp.body.saved_objects do ...`

Or, if the intent of the test is to only test the `managed` property, we can be even more lazy and just `map(obj => { id, managed})` and only assert against those values.